core/src/main/java/com/netflix/conductor/core/orchestration/ExecutionDAOFacade.java

```diff
@@ -230,7 +230,11 @@ public String updateWorkflow(Workflow workflow) {
                 workflow.getTasks().forEach(indexDAO::asyncIndexTask);
             } else {
                 indexDAO.indexWorkflow(workflow);
-                workflow.getTasks().forEach(indexDAO::indexTask);
+                /*
```

Comment:
Can remove this comment section here since the comments below in `updateTask` covers this.

Reply:
@apanicker-nflx, I removed the comments in my repo.